src/commissaire/bus/__init__.py

```diff
@@ -68,7 +68,24 @@ def __init__(self, message, model=None, data={}):
         if model is not None:
             data['model_json_data'] = model.to_dict_safe(),
             data['model_type_name'] = model.__class__.__name__
-        super(StorageLookupError, self).__init__(message, code, data)
+        super().__init__(message, code, data)
+
+
+class ContainerManagerError(RemoteProcedureCallError):
+    """
+    Exception class for container manager errors.
+    """
+    def __init__(self, message, data={}):
+        """
+        Creates a ContainerManagerError instance.
+
+        :param message: Error message
+        :type message: str
+        :param data: Additional error data
+        :type data: dict
+        """
+        code = C.JSONRPC_ERRORS['CONTAINER_MANAGER_ERROR']
+        super().__init__(message, code, data)
```

Comment:
Not for this PR, but we should probably update this at some point. Python 2 uses ``message`` while 3 moved to ``args``.